src/components/BarChart/BarChart.jsx

```diff
@@ -335,6 +358,46 @@ const BarChart = createClass({
 			? yAxisTooltipDataFormatter
 			: yAxisFinalFormatter;
 
+		if (_.isEmpty(data) || width < 1 || height < 1) {
```

Comment:
I think you need to add `isLoading` to this condition also, so that it's possible to show loading even if the chart already has data.